cmd/clusterctl/client/config/cert_manager.go

```diff
@@ -19,7 +19,7 @@ package config
 // CertManager defines cert-manager configuration.
 type CertManager interface {
 	// URL returns the name of the cert-manager repository.
-	// If empty, "https://github.com/cert-manager/cert-manager/releases/latest/cert-manager.yaml" will be used.
+	// If empty, "https://github.com/cert-manager/cert-manager/releases/{DefaultVersion}/cert-manager.yaml" will be used.
```

Comment:
Is this change related to the goproxy changes?

Reply:
Not directly, but was included in the first PR
Due to the first commit which is required in order to apply the second one. 

Original change was at #7192